syncd/syncd_counters.cpp

```diff
@@ -136,10 +134,10 @@ void endCountersThread()
 
     if (g_countersThread != NULL)
     {
-        SWSS_LOG_NOTICE("counters thread join");
+        SWSS_LOG_INFO("counters thread join");
 
         g_countersThread->join();
     }
 
-    SWSS_LOG_NOTICE("counters thread ended");
+    SWSS_LOG_INFO("counters thread ended");
```

Comment:
I think they are more code related logs that end users are  not going to care about. So I move it to INFO.